interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -39,21 +40,20 @@ function postcalendar_admin_modifyconfig($msg = '', $showMenu = true)
 
     $output->SetInputMode(_PNH_VERBATIMINPUT);
 
-    $header = "<html><head><title>" . xlt("Calendar") . "</title></head>";
-    $header .= <<<EOF
-	
-	<body bgcolor=
-EOF;
-    $header .= '"' . attr($GLOBALS['style']['BGCOLOR2']) . '">';
+    $header = "<html><head><title>" . xlt("Calendar") . "</title>";
+    $header .= Header::setupHeader('', false)  . '</head><body>';
+
     $output->Text($header);
-    if ($showMenu) {
-        $output->Text(postcalendar_adminmenu());
-    }
 
     if (!empty($msg)) {
-        $output->Text('<center><div style="padding:5px; border:1px solid green; background-color: lightgreen;">');
+        $output->Text(postcalendar_adminmenu("clearCache"));
+        $output -> Text('<div class="alert alert-success ml-1 mr-1 text-center" role="alert">');
```

Comment:
Can use `mx-1` here inplace of `ml-1 mr-1`

Reply:
Good ideas :)